Resources/config/yui.yml

```diff
@@ -333,13 +333,20 @@ system:
                     requires: ['ez-dashboardblockbaseview', 'ez-asynchronousview', 'event-tap', 'event-outside']
                     path: "%ez_platformui.public_dir%/js/views/dashboard/ez-dashboardblockasynchronousview.js"
                 ez-dashboardblockallcontentview:
-                    requires: ['ez-dashboardblockasynchronousview', 'dashboardblockallcontentview-ez-template']
+                    requires:
+                        - 'ez-dashboardblockasynchronousview'
+                        - 'dashboardblockallcontentview-ez-template'
+                        - 'ez-draftconflictbutton'
```

Comment:
it's not a matter of *button* anymore, I would name the module `ez-draftconflict` and the corresponding extension `Y.eZ.DraftConflict`